### ISSUE TITLE
Horizontal scroll hides the code

### ISSUE THREAD

**Issue by isaacdeveloper** [zorgzerg/brackets-minimap#29]
Hi, there is an error in horizontal scroll because when I scroll right, minimap hides the code and I can't see.

![screen](https://cloud.githubusercontent.com/assets/12492631/10820594/4e775c0c-7e4e-11e5-9482-9fb45a806542.gif)

**Comment by zorgzerg** [comment]
Please, adjust width and transparency in your minimap (Shift + MouseWheel). It will be good for you.

**Comment by isaacdeveloper** [comment]
Yes, but I think It should adjusted the overflow with minimap. It would very good. You know what I mean?

**Comment by ckubitza** [comment]
I also think this extension does not work very well with long text lines. You can not click on the text behind the minimap, I often find this confusing when selecting something or want to insert some text. It would be nice if we can optionaly place the minimap in a separate sidebar so it doesn't hide the content. Another advantage would be that the normal scroll bar stays visible and then you could better see the search hit markers. But beside of these small issues i find this extension great :)
